tool/tctl/common/plugin/awsic.go

```diff
@@ -87,17 +89,41 @@ func (a *awsICInstallArgs) validate(ctx context.Context, log *slog.Logger) error
 	return nil
 }
 
-func (a *awsICInstallArgs) validateSystemCredentialInput() error {
-	if !a.useSystemCredentials {
-		return trace.BadParameter("--use-system-credentials must be set. The tctl-based AWS IAM Identity Center plugin installation only supports AWS local system credentials")
+type integrationGetter interface {
+	GetIntegration(ctx context.Context, name string) (types.Integration, error)
+}
+
+func (a *awsICInstallArgs) validateCredentialInput(ctx context.Context, integrations integrationGetter, log *slog.Logger) error {
+	if a.useSystemCredentials {
+		if a.assumeRoleARN == "" {
+			return trace.BadParameter("--assume-role-arn must be set when --use-system-credentials is configured")
+		}
+
+		if a.oidcIntegration != "" {
+			return trace.BadParameter("--oidc-integration must not be set when --use-system-credentials is configured")
+		}
+
+		if _, err := awsutils.ParseRoleARN(a.assumeRoleARN); err != nil {
+			return trace.Wrap(err)
+		}
+		return nil
 	}
 
-	if a.assumeRoleARN == "" {
-		return trace.BadParameter("--assume-role-arn must be set when --use-system-credentials is configured")
+	if a.oidcIntegration == "" {
+		return trace.BadParameter("--oidc-integration must be set when --no-use-system-credentials is configured")
 	}
 
-	if _, err := awsutils.ParseRoleARN(a.assumeRoleARN); err != nil {
-		return trace.Wrap(err)
+	if a.assumeRoleARN != "" {
+		return trace.BadParameter("--assume-role-arn must not be set when --no-use-system-credentials is configured")
+	}
+
+	_, err := integrations.GetIntegration(ctx, a.oidcIntegration)
+	if err != nil {
+		if !a.forceOIDCIntegration {
```

Comment:
Why we need this flow ? 

What is the use case of creating AWS IC with OIDC creds without OIDC integration ?